torchinductor/codegen/triton.py

```diff
@@ -435,6 +442,8 @@ def reduction(self, name, dtype, reduction_type, index, value):
         )
         sizes = [f"{tree.prefix.upper()}BLOCK" for tree in self.range_trees]
         sizes[-1] = "1"
+        if reduction_type == "any":
+            reduction_type = "max"
```

Comment:
Add a comment to explain this?

Reply:
I can add one.  Triton doesn't have any().